sycl/include/CL/sycl/detail/pi.h

```diff
@@ -440,7 +440,17 @@ typedef enum {
 
 typedef enum {
   // Device-specific value opaque in PI API.
-  PI_MEM_ADVISE_UNKNOWN
+  PI_MEM_ADVISE_UNKNOWN,
```

Comment:
NIT: to follow PI API notation and repeat enum typename:
```suggestion
  PI_MEM_ADVIcE_UNKNOWN,
```